include/boost/url/pct_string_view.hpp

```diff
@@ -152,52 +152,34 @@ class pct_string_view final
 
         @par Postconditions
         @code
-        this->data() == s
+        this->data() == string_view(s).data()
         @endcode
 
         @par Complexity
-        Linear in `std::strlen(s)`.
+        Linear in `string_view(s).size()`.
 
         @par Exception Safety
         Exceptions thrown on invalid input.
 
         @throw system_error
         The string contains an invalid percent encoding.
 
-        @param s The string to construct from.
-    */
-    pct_string_view(
-        char const* s)
-        : pct_string_view(
-            string_view(s))
-    {
-    }
-
-    /** Constructor
-
-        The newly constructed string will
-        reference the specified character
-        buffer. Ownership is not transferred.
-
-        @par Postconditions
-        @code
-        this->data() == s
-        @endcode
-
-        @par Complexity
-        Linear in `s.size()`.
-
-        @par Exception Safety
-        Exceptions thrown on invalid input.
-
-        @throw system_error
-        The string contains an invalid percent encoding.
+        @tparam StringLike A type convertible to @ref string_view
 
         @param s The string to construct from.
     */
-    template <class Allocator>
+    template<
+        class StringLike
```

Comment:
Lets call it "String" so users don't get confused... I copied from cppreference which probably wasn't the best.

Reply:
Done